examples/apps/dashboard/dashboard.py

```diff
@@ -0,0 +1,227 @@
+        <img src=\"{member['Image']}\" title=\"{member['Name']}\"
+             style=\"
+                width: {image_size};
+                height: {image_size};
+                border-radius: 50%;
+                border: 2px solid white;
+                position: relative;
+                left: {idx * int(overlap_offset.replace('px', ''))}px;
+                z-index: {len(members)-idx};
+                transition: transform 0.2s;
+             \"
+             onmouseover=\"this.style.transform='scale(1.2)';this.style.zIndex='{len(members)+1}'\"
+             onmouseout=\"this.style.transform='scale(1)';this.style.zIndex='{len(members)-idx}'\"
+        >
+        '''
+    wrapper_html = f'''
+    <div style=\"display: flex; align-items: center;\">
+        {images_html}
+    </div>
+    '''
+    return wrapper_html
+
+def to_styles_projects_table(data: pd.DataFrame):
+    data["Company"]=data.apply(generate_company_html, axis=1)
+    data["Completion"]=data.apply(generate_progress_bar, axis=1)
+    data["Members"]=data.apply(generate_member_images, axis=1)
+    data = data.drop(columns=["CompanyImage"])
+    styled_df = data.style
+    styled_df = (
+        styled_df.hide(axis="index")
+        .set_table_styles(
+            [
+                {
+                    "selector": "th",
+                    "props": [
+                        ("background-color", "white"),
+                        ("font-weight", "bold"),
+                        ("border-bottom", "1px solid black"),
+                        ("text-align", "left"),
+                    ],
+                },
+                {
+                    "selector": "td",
+                    "props": [
+                        ("padding", "10px"),
+                        ("background", "white"),
+                        ("border-top", "1px solid black"),
+                    ],
+                },
+            ]
+        )
+        .set_properties(
+            **{"text-align": "left"},
+        )
+    )
+
+    return styled_df
+
+
+with pn.config.set(sizing_mode="stretch_width"):
+    web_site_views = pn.Column(pn.Column(
+        "#### Website Views\n\nLast Campaign Performance",
+        pn.pane.ECharts(
+            get_website_views_config(),
+            height=250,
+            margin=(0, 5),
+        ),
+        pn.Spacer(sizing_mode="stretch_height"),
+        pmu.Divider(margin=(10,10,-5,10)),
+        last_update("campaign sent 2 days ago"),
+    ), styles=PAPER_STYLES, height=400, margin=10, )
+
+    daily_sales = pn.Column(
+        "#### Daily Sales\n\n**+15%** increase in todays sales",
+        pn.pane.ECharts(
+            get_daily_sales_config(), height=250, margin=(0, 5),
+        ),
+        pn.Spacer(sizing_mode="stretch_height"),
+        pmu.Divider(margin=(10,10,-5,10)),
+        last_update("updated 4 min ago"),
+        styles=PAPER_STYLES,
+        margin=10,
+        height=400,
+    )
+    completed_tasks = pn.Column(
```

Comment:
These should simply user `Paper` instead of styling a Column like `Paper`.

Reply:
I looks like `Paper` is working now. I will do more testing.

But now something else is wrong

![image](https://github.com/user-attachments/assets/a85ab4d8-2688-4895-b9a0-adfd6ecbf2dc)

I've git pulled the latest and recompiled the js.

Reply:
I removed the build folder and rebuilt. On reload it shows the error for some 100 ms and then reloads

https://github.com/user-attachments/assets/c2dc7df0-e39d-4784-b958-6628ce610fb1

Reply:
This is the line that fails

![image](https://github.com/user-attachments/assets/21a96355-4923-4cfe-9aa6-fe2e8dd679f4)

Reply:
I also see this version where the error shows. But when I resize the window it updates again without error.

https://github.com/user-attachments/assets/704fa28b-ff83-4202-893c-94eea4ba3cf8

Reply:
Fixed with panel 1.6.3rc1. But then a new issue is introduced - overflow/ scrollbars.

Now scrollbars are always added.

![image](https://github.com/user-attachments/assets/58d51122-0206-41e2-a0f1-b4253f7c3e2c)

Reply:
But now scrollbars show up way to easily.

![image](https://github.com/user-attachments/assets/28719029-a2f6-4993-98bd-854eb0031195)

There should never be a horizontal scrollbar as things should stretch. Only a vertical scrollbar should sometimes be there.